monitor/utils/events.js

```diff
@@ -34,7 +36,17 @@ const web3Foreign = new Web3(foreignProvider)
 
 const { getBlockNumber } = require('./contract')
 
+const cacheFilePath = './cachedEvents.json'
```

Comment:
No, the only thing to be aware of, is that this file is not located under `/mono/monitor/responses`, so that docker won't keep it between different runs